Applications/FileManager/DirectoryView.cpp

```diff
@@ -108,8 +110,32 @@ void DirectoryView::set_view_mode(ViewMode mode)
     ASSERT_NOT_REACHED();
 }
 
+void print_path_history(int index, Vector<String> path_history)
+{
+    if (path_history.size() > 0) {
```

Comment:
Could just print this unconditionally (without the if), wouldn't do much harm :)

Also, I generally prefer:

    for (const auto& path : m_path_history) {
    }

... for iteration. Avoids signedness warnings, too :)